.circleci/config.yml

```diff
@@ -40,7 +40,8 @@ jobs:
       - run:
           name: Run tests - opencensus
           command: |
-            nox -f nox.py
+            pip install --upgrade nox
```

Comment:
This is a workaround to the preinstalled nox. We don't need this once we move to Travis.